tests/test_secret_scrubbing.py

```
@@ -635,3 +641,82 @@ def test_logfire_token_prefix_scrubbing(exporter: TestExporter):
             }
         ]
     )
+
+
+def test_disabled_patterns(exporter: TestExporter, config_kwargs: dict[str, Any]):
+    logfire.configure(scrubbing=logfire.ScrubbingOptions(disabled_patterns=['session']), **config_kwargs)
+
+    logfire.info('hi', session_id='abc123', password='hunter2')
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "password"

(S106)